### PR TITLE
Pptp 1158 enrolment failure page

### PR DESCRIPTION
**Pull request by RyanKellyHMRC**
We ensure a discrete URL is used to aid Google Analytics failure metrics capture

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [x] Required Environment Config has been amended/added - N/A
 - [x] Links to dependencies have been included (BE/FE work) - N/A
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [x] No Accessibility errors/warnings reported by Wave - N/A

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -33,6 +33,10 @@ date.month = Month
 date.year = Year
 date.empty.error = Error: {0} cannot be empty
 
+error.title = Sorry, there is a problem with the service
+error.detail1 = Try again later.
+error.detail2 = We saved your answers. They will be available for 30 days.
```
Comment (SteveSugden) [inline conf/messages.en:38]: "We saved your answers". This is not true in the case of enrolment failure, is it?

Comment (RyanKellyHMRC) [reply conf/messages.en:38]: We have introduced a new error page which does not mentioned the saving of answers.

File: test/uk/gov/hmrc/plasticpackagingtax/registration/views/ErrorNoSavePageSpec.scala [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    "contain heading" in {
+      view.select("h1").text() mustBe messages("error.title")
+    }
+
+    "contain detail" in {
+      val detail = view.select("p").text()
+      detail must include(messages("error.detail1"))
+      detail must not include (messages("error.detail2"))
```
Comment (SteveSugden) [inline test/uk/gov/hmrc/plasticpackagingtax/registration/views/ErrorNoSavePageSpec.scala:53]: 👍

**Review by SteveSugden — APPROVED** [review]
LGTM

**Pull request merged**